### ISSUE TITLE
Licensing?

### ISSUE THREAD

**Issue by hyandell** [qdot/libappindicator-rs#14]
I noticed the README stating the following:

```
tl;dr GNU LGPL v2.1, GNU LGPL v3 (Following license of libappindicator)

Licensed under the LGPL License, Version 3.0 (the "License");
you may not use this file except in compliance with the License.
```

The mention of 2.1 but then only saying 3.0 in the piece under the licensing confused me.

Is that indicating that users may use the code under either LGPL-2.1 OR LGPL-3.0, or is it indicating something else?

**Comment by qdot** [comment]
Yeah it's dual licensed under LGPL 2.1/3, following libappindicator's license. I'll try to clear that up in the text at some point.

**Comment by jxs** [comment]
Hi there, since [`libappindicator-sys` dynamic links against `libappindicator`](https://github.com/qdot/libappindicator-sys/blob/master/build.rs#L29) is there a reason for it to be LGPL?   Cause when one imports `libappindicator-rs` via rust it is static linked by `cargo` and therefore one can't write proprietary code using `libappindicator-rs` even if `libappindicator` is dynamic linked, and therefore abiding by [LGL license](http://www.gnu.org/licenses/gpl-faq.html#LGPLStaticVsDynamic) am I right?
thanks
